src/cpu/dynarmic_wrapper/build.rs

```diff
@@ -67,11 +73,16 @@ fn main() {
             .join("build/externals/fmt")
             .join(build_type_windows()),
     );
-    link_lib(if cfg!(debug_assertions) {
-        "fmtd"
+    // fmtd isn't on Linux
+    if !cfg!(target_os = "linux") {
+        link_lib(if cfg!(debug_assertions) {
+            "fmtd"
+        } else {
+            "fmt"
+        });
     } else {
-        "fmt"
-    });
+        println!("cargo:rustc-link-lib=dylib=fmt");
```

Comment:
Does fmt have to be dynamic? Does that actually work?

Reply:
Yup, there is no static library for fmt, only a shared library.

Reply:
Oh I have a guess as to what's going on: maybe we need to explicitly set the `DYNARMIC_USE_BUNDLED_EXTERNALS` cmake flag. On my macOS and Windows machines, I guess I just don't have any of those external libraries installed system-wide, but you have some of them, so that this works at all is an accident of our build environments.

Reply:
Specifically, what I think is going on is:

* On my machines, I have none of those libraries installed system-wide, so dynarmic falls back to using its bundled versions and the static libraries need to be linked.
* On your machine, you have fmt installed system-wide, so dynarmic doesn't build its bundled static version of _that_ library, but does build them for the rest.

I've been assuming this bundled approach the whole time, but I didn't realise it was environment-dependent.

Reply:
I guess we could have the “static” feature control this, but I don't think it's worth the extra complexity for now. Would you be okay with setting `DYNARMIC_USE_BUNDLED_EXTERNALS` always and then removing the static feature for this package?

Reply:
Should we be setting that flag for static builds?

Reply:
That would be the idea, though my preference would be to do it always and never attempt dynamic linking for the dynarmic dependencies, at least for now.